man/derive_param_computed.Rd

```diff
@@ -179,10 +186,29 @@ derive_param_computed(
     AVAL = (AVAL.SYSBP + 2 * AVAL.DIABP) / 3,
     PARAMCD = "MAP",
     PARAM = "Mean Arterial Pressure (mmHg)",
-    AVALU = "mmHg"
+    AVALU = "mmHg",
+    ADT = ADT.SYSBP
   )
 )
 
+Example 1b: Using option `keep_nas = TRUE` to derive MAP in the case where some/all values
+of a variable used in the computation are missing
+
```

Comment:
Is there data that has NAs in it?  I don't see it here?

Reply:
`ADTF` is all `NA` and it's one of the variables inside `set_values_to` in the second example (1b)

Reply:
oh right - okay I thought it was the specific calculation for doing MAP that needed NAs, but looking at issue I see I was misunderstanding.  

I think you need to add a # in your comment for 1b for the checks to pass.

Reply:
ah yes of course, thanks